source/ecommercestacks/magento/magento2/varnish/varnish.md

```diff
@@ -267,7 +267,10 @@ if (resp.status == 750) {
     }
 ```
 
-Under vcl_synth. Varnish will need a reload for this to take efect.
+Varnish will need a reload for this to take effect.
+```bash
+varnishd -C -f /etc/varnish/default.vcl && service varnish reload
```

Comment:
Would be tempted to amend this to 
```systemctl reload varnish ``` 
..for future proofing. Of course, the whole doc will have to be replaced by that point.